tests/test_self.py

```diff
@@ -100,6 +100,24 @@ def _configure_lc_ctype(lc_ctype: str) -> Iterator:
             os.environ[lc_ctype_env] = original_lctype
 
 
+@contextlib.contextmanager
```

Comment:
This could probably even go in ``tests/conftest.py`` to be used everywhere ? I don't know how frequent it is. But we might want to do that in another MR, this one need to be merged 😄